samples/package.json

```diff
@@ -16,17 +20,10 @@
   "devDependencies": {
     "@google-cloud/nodejs-repo-tools": "^3.0.0",
     "mocha": "^5.2.0",
-    "doctoc": "^1.3.1",
-    "proxyquire": "^2.0.0",
-    "sinon": "^7.0.0"
-  },
-  "nyc": {
-    "exclude": [
-      "**/*.test.js"
-    ]
+    "doctoc": "^1.3.1"
   },
   "scripts": {
     "doctoc": "doctoc README.md */README.md",
-    "test": "mocha system-test/*.test.js --timeout=600000"
+    "test": "mocha system-test --timeout=60000"
```

Comment:
60 seconds might be problematic.  Table create / modify / delete took 20+ seconds at some points.